pom.xml

```diff
@@ -280,6 +280,11 @@
                 <artifactId>jeval</artifactId>
                 <version>0.9.4</version>
             </dependency>
+            <dependency>
```

Comment:
Increment the version for the entire slacker codebase since you are changing the interface definitions for Action, Endpoint, Collector, etc.
So it doesn't break other external code that already relies on the current version.